src/views/AccessDenied/AccessDenied.js

```diff
@@ -79,6 +79,7 @@ const AccessDenied = ({ application }) => {
               <Grid item>
                 <FormControl className={classes.formControl}>
                   <Select
+                    variant="standard"
```

Comment:
I thought all inputs must be _outlined_ and not _standart_ :laughing:

Reply:
At this time, I have made parameters looking the actual theme. We will probably change them as outlined during the scenario parameters grid refacto.